tamr_toolbox/utils/version.py

```diff
@@ -1,31 +1,36 @@
 """Tasks related to the version of Tamr instances"""
-from tamr_unify_client import Client
-from typing import Callable
-import json
 import inspect
-
+import json
 import logging
+import warnings
+from collections.abc import Callable
+from typing import List, Optional
+
+from packaging.version import parse
+from tamr_unify_client import Client
 
 LOGGER = logging.getLogger(__name__)
+logging.captureWarnings(True)
 
 
 def current(client: Client) -> str:
-    """Gets the version of Tamr for provided client
+    """
+    Gets the version of Tamr for provided client
 
     Args:
         client: Tamr client
 
     Returns:
         String representation of Tamr version
-
     """
     url = "/api/versioned/service/version"
     response = client.get(url).successful()
     return json.loads(response.content)["version"]
 
 
 def _as_float(version: str) -> float:
```

Comment:
Parse, or client.version? I haven't come across parse before

Reply:
Your code is using `packaging.version.parse` to convert a Tamr version string to an ordering-safe object.  I mean we should point people towards using that approach over `_as_float` (even though it is "private" it would be nice to officially suggest an alternative).

Reply:
Ah OK, with you and completely agree.